core/trino-web-ui/src/main/resources/webapp-preview/vite.config.ts

```diff
@@ -9,7 +10,7 @@ export default defineConfig((mode: ConfigEnv) => {
   const baseUrl = env.VITE_BASE_URL
   return {
     base: '/ui/preview',
-    plugins: [react()],
+    plugins: [react(), monaco({features: [], languages: ['sql', 'java']})],
```

Comment:
@koszti Is Java used for stacktraces?

Reply:
Yes we can!

Reply:
can you please capture this wisdom with a code comment?